raster/r.random.cells/init.c

```diff
@@ -73,8 +73,8 @@ void Init()
     G_debug(1, "(CellCount):%d", CellCount);
 
     sscanf(Distance->answer, "%lf", &MaxDist);
-    if (MaxDist < 0.0)
-	G_fatal_error(_("Distance must be >= 0.0"));
+    if (!(MaxDist > 0.0))
```

Comment:
To me `MaxDist <= 0.0` is more readable, requires less thinking :sweat_smile:

Reply:
@metzm do you have any opinion on this suggested change?